src/api.rs

```diff
@@ -80,6 +80,13 @@ pub struct Children {
   pub page: usize,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct ChildInscriptions {
+  pub inscriptions: Vec<ChildInscriptionRecursive>,
```

Comment:
```suggestion
  pub children: Vec<ChildInscriptionRecursive>,
```
I don't have strong feelings about this, but from an end user (javascript dev) experience it could make the code clearer if it's called `children` instead of `inscriptions`. I'll refer to you and @owenstrevor

Reply:
It makes no difference to me what the field is called. I checked with Trevor and he's fine with it being changed like you suggested. Here's an out of context screenshot as fake proof of that.

![Screenshot_20240527_074131_Discord](https://github.com/ordinals/ord/assets/49558347/3dee5d10-85a7-47ed-9f1f-dfcb7a9799a5)

See? He's fine with it! I'll make a new commit ASAP.